lib/index.js

```diff
@@ -72,6 +75,31 @@ internals.reformat = function () {
     }
 };
 
+internals.extendOutput = function (obj) {
+
+    // Underscore _.isObject
+    var isObject = function(obj){return (typeof obj === 'function' || typeof obj === 'object' && !!obj)};
+
+    // Underscore _.extend
+    var extend = function(object) {
+        if (!isObject(object)) return object;
+        var source, prop;
+        for (var i = 1, length = arguments.length; i < length; i++) {
+          source = arguments[i];
+          for (prop in source) {
+            if (hasOwnProperty.call(source, prop)) {
+                object[prop] = source[prop];
+            }
+          }
+        }
+        return object;
+    };
+
+    extend(this.output.payload, obj)
```

Comment:
Yes, Hoek.merge would work here. This is actually copy of underscore's source code, but it makes sense chaning it to Hoek, in line with the library.